plugins/source/googleads/client/context.go

```diff
@@ -13,6 +13,9 @@ const (
 
 func (c *Client) OutgoingContext(ctx context.Context) context.Context {
 	ctx = c.addDeveloperToken(ctx)
+	if len(c.ManagerID) == 0 {
```

Comment:
If it's consistently there then why add it?

Reply:
Well, it was me who implemented this initially.
If someone then updates the code without this knowledge, they might end up setting the `login-customer-id` to `""`, which, in its turn, may produce unexpected API response.
Better err on the safe side.